src/commands/dev/dev.cjs

```diff
@@ -222,10 +222,11 @@ const startFrameworkServer = async function ({ settings }) {
     }
 
     stopSpinner({ error: false, spinner })
-  } catch {
+  } catch (error_) {
     stopSpinner({ error: true, spinner })
-    log(NETLIFYDEVERR, `Netlify Dev could not connect to localhost:${settings.frameworkPort}.`)
+    log(NETLIFYDEVERR, `Netlify Dev could not start or connect to localhost:${settings.frameworkPort}.`)
     log(NETLIFYDEVERR, `Please make sure your framework server is running on port ${settings.frameworkPort}`)
+    error(error_)
```

Comment:
This is actually unrelated and was for some debugging I did, but it actually makes sense. We usually land here when `wait-port` throws, so it is good to actually see the error that happened.